log/logback.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In this case, this file needs to be placed in the src/main/resources folder, since it's used by the build at runtime